docs/dev/task_failure_recovery.zh.md

```diff
@@ -260,4 +265,49 @@ env.setRestartStrategy(RestartStrategies.noRestart())
 这对于启用了 checkpoint 的流处理程序很有帮助。
 如果没有定义其他重启策略，默认选择固定延时重启策略。
 
+## 故障恢复策略
+
+Flink 支持多种不同的故障恢复策略，该策略需要通过 Flink 配置文件 `flink-conf.yaml` 中的 *jobmanager.execution.failover-strategy*
+配置项进行配置。
+
+<table class="table table-bordered">
+  <thead>
+    <tr>
+      <th class="text-left" style="width: 50%">故障恢复策略</th>
+      <th class="text-left">jobmanager.execution.failover-strategy 配置值</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+        <td>全图重启</td>
+        <td>full</td>
+    </tr>
+    <tr>
+        <td>基于 Region 的局部重启</td>
+        <td>region</td>
+    </tr>
+  </tbody>
+</table>
+
+### 全图重启故障恢复策略
+
+在全图重启故障恢复策略下，Task 发生故障时会重启作业中的所有 Task 进行故障恢复。
+
+### 基于 Region 的局部重启故障恢复策略
+
+本策略会以 Region 为粒度来决定需要重启的 Task。
+
+此处 Region 指以 Pipelined 形式进行数据交换的 Task 集合。
+- DataStream 和 流式 Table 作业的所有数据交换都是 Pipelined 形式的。
+- 批处理式 Table 作业的所有数据交换都是 Batch 形式的。
+- DataSet 作业中的数据交换形式会根据 [ExecutionConfig]({{ site.baseurl }}/dev/execution_configuration.html) 
```

Comment:
Use the Chinese version link which is start with "zh": `({{ site.baseurl }}/zh/dev/execution_configuration.html) `